scripts/install_python_prod_dependencies.py

```diff
@@ -95,6 +107,12 @@ def normalize_python_library_name(library_name: str) -> str:
     # scripts/install_python_prod_dependencies_test.py to ensure that all
     # library names in the requirements files are distinct when normalized.
     library_name = re.sub(r'\[[^\[^\]]+\]', '', library_name)
+    if any(
+            library_name.startswith('%s%s' % (prefix, '-'))
+            for prefix in LIBRARY_PREFIXES_WITH_INITIAL_PERIOD):
+        library_name_parts = library_name.split('-')
+        library_name = '%s.%s' % (
+            library_name_parts[0], '-'.join(library_name_parts[1:]))
```

Comment:
@seanlip Do we need to test this code?

Reply:
Good catch, thanks -- done! Added a test.